pytest.ini

```diff
@@ -9,7 +9,7 @@ addopts =
     # show values of the local vars in errors:
     --showlocals
     # coverage reports
-    --cov=aiohttp_admin/ --cov=tests/ --cov-report term
+    --cov --cov-report term
```

Comment:
I also like adding `-p pytest_cov` to force pytest to pre-load this specific plugin early (this helps with xdist sometimes, because it's activated well before any of the project code is imported).

Reply:
I think `--cov-report term` is the default. So it might be unnecessary.